src/condor_utils/file_transfer.cpp

```diff
@@ -4718,6 +4726,47 @@ MyString FileTransfer::GetSupportedMethods() {
 	return method_list;
 }
 
+int FileTransfer::InitializeJobPlugins(const ClassAd &job, CondorError &e, StringList &infiles)
+{
+	if ( ! I_support_filetransfer_plugins || ! plugin_table) {
+		return 0;
+	}
+
+	std::string job_plugins;
+	if ( ! job.LookupString(ATTR_TRANSFER_PLUGINS, job_plugins)) {
+		return 0;
+	}
+
+	StringTokenIterator plugins(job_plugins, 100, ";");
+	for (const char * plug = plugins.first(); plug != NULL; plug = plugins.next()) {
```

Comment:
I think this is easier to read:
```suggestion
	for (auto plug = plugins.first(); plug != NULL; plug = plugins.next()) {
```